lib/backends/sdc/containers.js

```diff
@@ -1507,15 +1507,28 @@ function buildVmPayload(opts, container, callback) {
         });
     }
     if (container.Labels) {
-        Object.keys(container.Labels).forEach(function _labelToTag(key) {
-            if (key === PACKAGE_SELECTION_LABEL) {
+        labelKeys = Object.keys(container.Labels);
+        for (i = 0; i < labelKeys.length; i++) {
+            labelKey = labelKeys[i];
+            if (labelKey === PACKAGE_SELECTION_LABEL) {
                 // PACKAGE_SELECTION_LABEL is a special label. It gets consumed
                 // here and doesn't end up in the user's list. Instead it's used
                 // in getPackage().
-                return;
+                continue;
             }
-            payload.tags[common.LABELTAG_PREFIX + key] = container.Labels[key];
-        });
+            if (triton_tags.isTritonTag(labelKey)) {
+                try {
```

Comment:
By having these validateTritonTag* functions throw and catching in this `try {`, we lose the ability to distinguish between programmer errors and runtime errors. For example we've validated that we have a triton tag here, but later we'll do:

```
assert.func(validator, 'validator for tag ' + key);
```

or:

```
if (type === undefined) {
    throw new Error('Unrecognized special triton tag "' + key + '"');
}
```

which seem like they would only happen on programmer error. But they'll be caught in this `try {` rather than throwing an uncaught exception and generating a core. Is this fine? It might be, just wanted to point it out.